src/FFMpeg/FFProbe.php

```diff
@@ -213,25 +214,32 @@ public function streams($pathfile)
      *
      * @param array|ConfigurationInterface $configuration
      * @param LoggerInterface              $logger
-     * @param Cache                        $cache
+     * @param CacheItemPoolInterface       $cache
      *
      * @return FFProbe
      */
-    public static function create($configuration = array(), LoggerInterface $logger = null, Cache $cache = null)
+    public static function create($configuration = array(), LoggerInterface $logger = null, CacheItemPoolInterface $cache = null)
     {
         if (null === $cache) {
-            $cache = new ArrayCache();
+            $cache = new ArrayAdapter();
         }
 
         return new static(FFProbeDriver::create($configuration, $logger), $cache);
     }
 
     private function probe($pathfile, $command, $type, $allowJson = true)
     {
-        $id = sprintf('%s-%s', $command, $pathfile);
+        if (PHP_VERSION_ID < 72000) {
+            // symfony/cache:4.* does not accept slashes '/' in keys
+            $key = sprintf('%s-%s', $command, md5($pathfile));
+        } else {
+            $key = sprintf('%s-%s', $command, $pathfile);
+        }
```

Comment:
Hi @ekyna. I'm using your patch (this PR) in my project. I'm having issues here, I use PHP 7.4 with `symfony/cache:4`.
Do you think there's a better way to perform this check?

Reply:
@amiranagram Maybe we can always use hash function like md5() and remove the PHP version test ?

Reply:
I've already done that in my fork but I'm not sure if it will work with other versions of `symfony/cache`. Maybe a couple of tests will solve this.

https://github.com/AdaptechLLC/PHP-FFMpeg/commit/37db9819ef366bebe1edfb9e7bf669139641b3ef

Reply:
Changed this PR to always use md5().